lib/utils.js

```diff
@@ -110,8 +119,25 @@ function createFindExpression(j, args) {
  * @param args  function arguments
  * @returns {*}
  */
-function createFindAllExpression(j, args) {
-  args = args.length > 0 ? args.map(s => migrateSelector(j, s)) : [j.literal('*')];
+function createFindAllExpression(j, args, fileRoot) {
+  args = args.length > 0 ? args.map(s => migrateSelector(j, s, fileRoot)) : [j.literal('*')];
+  let newArgs = args.length > 0 ? args.map(s => migrateSelector(j, s, fileRoot)) : [j.literal('*')];
```

Comment:
Seems to be unused!

Reply:
🙈 you aren't supposed to see my debugging kludge. I'll remove it.